make/photon/jobservice/Dockerfile

```diff
@@ -1,13 +1,19 @@
 FROM photon:2.0
 
-RUN mkdir /harbor/ \
-    && tdnf install sudo -y >> /dev/null\
+RUN tdnf install sudo -y >> /dev/null\
```

Comment:
do we still need `sudo`?

Reply:
yes, `chown`, `chmod` are included in sudo package, we need these command while build the image